UPGRADE.rst

```diff
@@ -132,6 +114,20 @@ request to
 with the query parameters from the original link, presented as a URL-encoded form. See the file
 itself for more details.
 
+Updated Single Sign-on HTML Templates
+-------------------------------------
+
+The ``saml_error.html`` template was removed from Synapse and replaced with the
+``sso_error.html`` template. If your Synapse is configured to use SAML and a
+custom ``sso_redirect_confirm_template_dir`` configuration then the
+``saml_error.html`` template will need to be renamed to ``sso_error.html`` and
+updated:
```

Comment:
`sso_error.html` already exists right? As a user I may be a bit confused if I've branded both pages already, and I'm being told to "rename one to the other".

Reply:
It does, but if you were using SAML you wouldn't/shouldn't have been using it. I'll try to clarify this though!

Reply:
Looks great, thanks!